day2/es1/es1.qml

```diff
@@ -0,0 +1,45 @@
+import QtQuick 2.12
+import QtQuick.Window 2.12
+import QtQuick.Layouts 1.15
+
+Window {
+    id: mainWindow
+
+    width: 640
+    height: 480    
+    title: qsTr("Hello World")
+
+    Column {
+        spacing: 0
+
+        Rectangle {
+            id: windowArea
+
+            property var cardPrefab: Qt.createComponent("Card.qml");
+
+            function createCard(text) {
+                while (windowArea.cardPrefab.status !== Component.Ready){ }
```

Comment:
Eviterei di aggiungere questo tipo di attese attive, in generale non mi aspetto che una funzione il cui scopo è creare card verifichi se l'area di aggiunta è pronta, perchè do per scontato che nessuno cerchi di aggiungere card prima del tempo.

Reply:
Rimosso, intanto durante la review luca mi ha spiegato meglio come mai a quel punto è sicuramente ready, e mi ha anche suggerito di fare
```
            Component {
                id: cardComponent
                Card {
                }
            }
```
al posto di `property var cardPrefab: Qt.createComponent("Card.qml");`

Reply:
Perfetto; sì ho visto la modifica col Component :+1: